Sources/TokamakCore/MountedViews/MountedElement.swift

```diff
@@ -76,32 +76,45 @@ public class MountedElement<R: Renderer> {
   var typeConstructorName: String {
     switch element {
     case .app: fatalError("""
-      `App` values aren't supposed to be reconciled, thus the type constructor name is not stored \
-      for `App` elements. Please report this crash as a bug at \
-      https://github.com/swiftwasm/Tokamak/issues/new
-      """)
+    `App` values aren't supposed to be reconciled, thus the type constructor name is not stored \
+    for `App` elements. Please report this crash as a bug at \
+    https://github.com/swiftwasm/Tokamak/issues/new
+    """)
     case let .scene(scene): return scene.typeConstructorName
     case let .view(view): return view.typeConstructorName
     }
   }
 
   var mountedChildren = [MountedElement<R>]()
   var environmentValues: EnvironmentValues
+  unowned var parent: MountedElement<R>?
```

Comment:
I never remember what are the reasons to use `unowned` instead of `weak`, I guess the latter is always safer?
```suggestion
  weak var parent: MountedElement<R>?
```

Reply:
Unowned is for when the thing it references has a lifetime >= it’s own IIRC.